### PR TITLE
Pipfile and upgrade to DJ1.11

### PR DESCRIPTION
**Pull request by domdinicola**
I updated Django because when trying to lock the dependecies I get this error
Hint: try $ pipenv lock --pre if it is a pre-release dependency.
ERROR: ERROR: Could not find a version that matches django==1.9.13,>=1.11,>=1.5

### CODE REVIEW

**Comment by czue** [comment]
thanks! happy to see this direction. a few things that I think should also to get updated:

- requirements files should be removed, yeah?
- we should update readme and any other relevant docs with the new setup instructions.

**Comment by domdinicola** [comment]
@czue please check again

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+services:
+  - postgresql
+
+
+addons:
+  postgresql: "9.5"
+  apt:
+    packages:
+      - postgresql-9.5-postgis-2.4
```
Comment (czue) [inline .travis.yml]: Is this required?

Comment (domdinicola) [reply .travis.yml]: ops..  copy paste magic :)

**Comment by czue** [comment]
:tada:

**Pull request merged**